### PR TITLE
feat: take rate, multi-currency

### PR DESCRIPTION
**Pull request by boz**
* Allow leases to be priced in arbitrary denomination.
* Add gov-configurable default & denomination-specific "take rate".
* Add gov-configurable default & denomination-specific deposit amounts.

### CODE REVIEW

**Comment by troian** [comment]
just cut release with take pay [akash-api v0.0.17](https://github.com/akash-network/akash-api/releases/tag/v0.0.17)

File: app/app_configure.go [hunk 1]
```diff
@@ -297,7 +317,9 @@ func (app *AkashApp) akashInitGenesisOrder() []string {
 		transfertypes.ModuleName,
 		feegrant.ModuleName,
 		cert.ModuleName,
+		taketypes.ModuleName,
 		escrow.ModuleName,
+		take.ModuleName,
```
Comment (troian) [inline app/app_configure.go:322]: ✂️

File: app/app_configure.go [hunk 2]
```diff
@@ -83,15 +88,23 @@ func akashSubspaces(k paramskeeper.Keeper) paramskeeper.Keeper {
...
-
+	k.Subspace(take.ModuleName)
 	return k
 }
 
 func (app *AkashApp) setAkashKeepers() {
+
+	app.Keepers.Akash.Take = tkeeper.NewKeeper(
+		app.appCodec,
+		app.keys[escrow.StoreKey],
```
Comment (troian) [inline app/app_configure.go:99]: `take.StoreKey`

Comment (boz) [reply app/app_configure.go:98]: :hankey:

File: app/app_configure.go (hunk 1 above)
Comment (boz) [reply app/app_configure.go]: It complained that I didn't have it in there.  Maybe something else triggered it, will try.

Comment (boz) [reply app/app_configure.go]: ah, you pulled it already.  thanks!

**Review by troian — APPROVED** [review]

**Pull request merged**